frontend/lib/pages/authentication_service.dart

```diff
@@ -0,0 +1,101 @@
+import 'package:http/http.dart' as http;
+import 'dart:convert';
+
+class AuthenticationService {
+  static const String baseUrl = 'http://localhost:8080'; // Connecting to url
+
```

Comment:
Make sure to add function headers like this:
    /**
     * End point to get events on a certain date
     * @param month month of date
     * @param day day of date
     * @param year year of date
     * EXAMPLE http://localhost:8080/event/byDate/06/01/2024
     * @return JSON of all events on that date
     */
    @GetMapping("byDate/{month}/{day}/{year}")
    public List<Event> getByDate(@PathVariable Integer month,
                                 @PathVariable Integer day, @PathVariable Integer year) {
        return eventService.getEventsByDate(month,day,year);
    }